src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -39,12 +42,15 @@ def load_command_table(self, _):
     agent_pools_sdk = CliCommandType(
         operations_tmpl='azext_aks_preview.vendored_sdks.azure_mgmt_preview_aks.'
                         'operations._agent_pools_operations#AgentPoolsOperations.{}',
+        resource_type=ResourceType.MGMT_CONTAINERSERVICE,
         client_factory=cf_managed_clusters
     )
 
     openshift_managed_clusters_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.containerservice.v2018_09_30_preview.operations.'
+        operations_tmpl='azure.mgmt.containerservice.operations.'
```

Comment:
same here, this affects azure as well, you may want to add an entry in latest Profile map in _shared.py

Reply:
we already have added api versions to latest profile in _shared.py